test/e2e/create_test.go

```diff
@@ -61,6 +61,7 @@ func creationTestSuite(t *testing.T) {
 	t.Run("Certificates", testCertificates)
 	t.Run("Node Logs", testNodeLogs)
 	t.Run("Metrics validation", testMetrics)
+	t.Run("Dependent service reconcilation", testDependentServiceChanges)
```

Comment:
Won't this still run in the upgrade job, extending its runtime? Perhaps we can add a platform dependent skip for AWS